snap/README.md

```diff
@@ -52,48 +58,28 @@ $ sudo cp /snap/kata-containers/current/usr/share/defaults/kata-containers/confi
 $ $EDITOR /etc/kata-containers/configuration.toml
 ```
 
-## Integration with docker ##
-
-the path to the runtime provided by the Kata Containers snap image is
-`/snap/kata-containers/current/usr/bin/kata-runtime`, this runtime must be added to
-[dockerd][5] via `systemd` or `dockerd` configuration file.
+## Integration with docker and kubernetes
 
-`/etc/systemd/system/docker.service.d/runtime.conf`
-
-```ini
-[Service]
-ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/snap/kata-containers/current/usr/bin/kata-runtime --default-runtime=kata-runtime
-```
-
-or
-
-`/etc/docker/daemon.json`
-
-```json
-{
-	"default-runtime": "kata-runtime",
-	"runtimes": {
-		"kata-runtime": {
-			"path": "/snap/kata-containers/current/usr/bin/kata-runtime"
-		}
-	}
-}
-```
-
-after having added the new runtime, the service must be reloaded and restarted
-
-```
-$ sudo systemctl daemon-reload
-$ sudo systemctl restart docker
-```
+The path to the runtime provided by the Kata Containers snap image is
+`/snap/kata-containers/current/usr/bin/kata-runtime`, it should be used to
+run Kata Containers with [docker][9] and [kubernetes][10].
 
 ## Limitations
 
 The [miniOS image][2] is not included in the snap image as it is not possible for
-QEMU to open a guest RAM backing store on a read-only filesystem.
+QEMU to open a guest RAM backing store on a read-only filesystem, fortunately
```

Comment:
Lines 70-74 suggested rewrite (if I haven't changed the meaning): 
QEMU to open a guest RAM backing store on a read-only filesystem. Fortunately, you can start Kata Containers with a Linux initial RAM disk (initrd) that is included in the snap image. If you want to use the miniOS image instead of initrd, then a new configuration file can be [created](#configuring-kata-containers) and [configured][7].